api/net/Areas/Admin/Controllers/ContentReferenceController.cs

```diff
@@ -106,8 +106,23 @@ public IActionResult Update(ContentReferenceModel model)
     [SwaggerOperation(Tags = new[] { "ContentReference" })]
     public IActionResult Delete(ContentReferenceModel model)
     {
-        _service.DeleteAndSave(model.ToEntity());
+        _contentReferenceService.DeleteAndSave(model.ToEntity());
         return new JsonResult(model);
     }
+
+    /// <summary>
+    /// Find all the content ids for the specified 'uid'.
+    /// </summary>
+    /// <param name="uid"></param>
+    /// <returns></returns>
+    [HttpGet("content/ids")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(long[]), (int)HttpStatusCode.OK)]
+    [SwaggerOperation(Tags = new[] { "ContentReference" })]
+    public IActionResult FindContentId([FromQuery] string uid)
```

Comment:
New endpoint to find the content ids for the specified 'uid'.  It should only ever return one item, but it's possible to return many.